test/bitswap.js

```diff
@@ -134,9 +134,8 @@ describe('bitswap with DHT', () => {
   it('put a block in 2, get it in 0', (done) => {
     waterfall([
       (cb) => makeBlock(cb),
-      (block, cb) => setTimeout(() => cb(null, block), 1000),
       (block, cb) => nodes[2].bitswap.put(block, () => cb(null, block)),
-      (block, cb) => setTimeout(() => cb(null, block), 1000),
+      (block, cb) => setTimeout(() => cb(null, block), 400),
       (block, cb) => nodes[0].bitswap.get(block.cid, (err, blockRetrieved) => {
```

Comment:
The fix was adding the timeout, it seems that the propagation after the put takes more time on CI. I think this fix is "good enough" to move forward with this implementation, but we should work on setting up more integration style tests to make sure content is distributed well in the actual tests.